// _@vuepress_core@1.5.4@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "F:\\dhr-guide\\node_modules\\_@vuepress_core@1.5.4@@vuepress\\core\\lib\\client\\components\\GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-0a234648",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-0a234648").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-1267efeb",
    path: "/javaEE/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-1267efeb").then(next)
    },
  },
  {
    path: "/javaEE/index.html",
    redirect: "/javaEE/"
  },
  {
    name: "v-3ed79340",
    path: "/javaEE/javaEE-1.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3ed79340").then(next)
    },
  },
  {
    name: "v-857c44aa",
    path: "/javaSE/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-857c44aa").then(next)
    },
  },
  {
    path: "/javaSE/index.html",
    redirect: "/javaSE/"
  },
  {
    name: "v-73bade6a",
    path: "/springBoot/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-73bade6a").then(next)
    },
  },
  {
    path: "/springBoot/index.html",
    redirect: "/springBoot/"
  },
  {
    name: "v-1d954e60",
    path: "/javaSE/javaSE-1.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-1d954e60").then(next)
    },
  },
  {
    name: "v-a4dfd4fa",
    path: "/springCloud/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-a4dfd4fa").then(next)
    },
  },
  {
    path: "/springCloud/index.html",
    redirect: "/springCloud/"
  },
  {
    name: "v-2fe7e9e0",
    path: "/springBoot/springBoot-1.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2fe7e9e0").then(next)
    },
  },
  {
    name: "v-024afb14",
    path: "/%E5%88%86%E5%B8%83%E5%BC%8F/%E5%88%86%E5%B8%83%E5%BC%8F-1.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-024afb14").then(next)
    },
  },
  {
    path: "/分布式/分布式-1.html",
    redirect: "/%E5%88%86%E5%B8%83%E5%BC%8F/%E5%88%86%E5%B8%83%E5%BC%8F-1.html"
  },
  {
    path: "/分布式/分布式-1.html",
    redirect: "/%E5%88%86%E5%B8%83%E5%BC%8F/%E5%88%86%E5%B8%83%E5%BC%8F-1.html"
  },
  {
    name: "v-5ceb0ce6",
    path: "/springCloud/springCloud-1.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5ceb0ce6").then(next)
    },
  },
  {
    name: "v-7ae8a354",
    path: "/%E6%95%B0%E6%8D%AE%E5%BA%93/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7ae8a354").then(next)
    },
  },
  {
    path: "/数据库/",
    redirect: "/%E6%95%B0%E6%8D%AE%E5%BA%93/"
  },
  {
    path: "/%E6%95%B0%E6%8D%AE%E5%BA%93/index.html",
    redirect: "/%E6%95%B0%E6%8D%AE%E5%BA%93/"
  },
  {
    path: "/数据库/",
    redirect: "/%E6%95%B0%E6%8D%AE%E5%BA%93/"
  },
  {
    name: "v-571856fc",
    path: "/%E6%95%B0%E6%8D%AE%E5%BA%93/%E6%95%B0%E6%8D%AE%E5%BA%93-1.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-571856fc").then(next)
    },
  },
  {
    path: "/数据库/数据库-1.html",
    redirect: "/%E6%95%B0%E6%8D%AE%E5%BA%93/%E6%95%B0%E6%8D%AE%E5%BA%93-1.html"
  },
  {
    path: "/数据库/数据库-1.html",
    redirect: "/%E6%95%B0%E6%8D%AE%E5%BA%93/%E6%95%B0%E6%8D%AE%E5%BA%93-1.html"
  },
  {
    name: "v-3cc40274",
    path: "/%E5%88%86%E5%B8%83%E5%BC%8F/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3cc40274").then(next)
    },
  },
  {
    path: "/分布式/",
    redirect: "/%E5%88%86%E5%B8%83%E5%BC%8F/"
  },
  {
    path: "/%E5%88%86%E5%B8%83%E5%BC%8F/index.html",
    redirect: "/%E5%88%86%E5%B8%83%E5%BC%8F/"
  },
  {
    path: "/分布式/",
    redirect: "/%E5%88%86%E5%B8%83%E5%BC%8F/"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
